app/views/docs/view.html.erb

```diff
@@ -1,2 +1,49 @@
-
-<%= raw @result %>
+<section class="cli-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto border-bottom">
+  <div class="container cointaier-shrinked">
+    <h1 class="display-4 text-left text-success">
+      Docs
+      <button class="navbar-toggler float-right mt-4 d-md-none" type="button" data-toggle="collapse" data-target="#docs-sidebar" aria-controls="docs-sidebar" aria-expanded="false" aria-label="Toggle Docs Navigation">
+        <span><i class="fa fa-bars"></i></span>
+      </button>      
+    </h1>
+  </div>
+</section>
+<section class="pt-8 pt-md-11 bg-dark p-3 p-md-5 border-top">
+  <div class="container text-white">
+    <div class="row">
+        <div class="col-lg-3 col-md-3 col-sm-12 d-sm-flex bg-light pt-2 pl-0 collapse d-md-show navbar-collapse" id="docs-sidebar">
+          <div class="card bg-secondary">
+            <div class="card-body text-white">
+              <ul class="nav flex-column flex-nowrap overflow-hidden">                
+                <% directory_hash("#{Rails.root}/public/documentation")[:children].reverse_each do |item| %>
+                  <li class="nav-item">
+                    <% if item[:children].present? %>
```

Comment:
Also there is no need to have sub items "index", For Cloud Hosting, it links to cloud_hosting/index.md